### DISCUSSION TITLE
Progetto Esame open source Alessandro Manucci

### DISCUSSION THREAD

**Discussion by Dreadfulorca** [deneb-alpha/Lab_open_source-2023_2024#28] [Projects]
Parte Principale:
- Definizione di licenza
- Scopo di una licenza
- Approfondimento su licenze software
- Scelta della licenza che fa per te
- Installazione e utilizzo di Reuse

Parte facoltativa:
- Illustrazione di tutti i passaggi utilizzati per l'installazione e l'implementazione di Python e di Reuse

**Comment by deneb-alpha** [comment]
Ciao @Dreadfulorca,
la parte principale che hai descritto va bene. Se possibile, nel descrivere le licenze, cerca anche di citare alcuni progetti che utilizzano la licenza che stai citando e magari anche i vantaggi/svantaggi che un contributore può trovarsi ad affrontare a causa di una specifica licenza. Ad esempio, alcuni progetti associano una Contributor License Agreement (CLA)

**Comment by Dreadfulorca** [comment]
Salve profGrazie della precisazione, adesso apporterò le modifiche. Inviato da Posta per Windows Da: Marina LatiniInviato: domenica 14 gennaio 2024 19:41A: deneb-alpha/Lab_open_sourceCc: Dreadfulorca; MentionOggetto: Re: [deneb-alpha/Lab_open_source] Progetto Esame open source Alessandro Manucci (Discussion #28) Ciao @Dreadfulorca,la parte principale che hai descritto va bene. Se possibile, nel descrivere le licenze, cerca anche di citare alcuni progetti che utilizzano la licenza che stai citando e magari anche i vantaggi/svantaggi che un contributore può trovarsi ad affrontare a causa di una specifica licenza. Ad esempio, alcuni progetti associano una Contributor License Agreement (CLA)—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
